src/rpc.rs

```diff
@@ -121,13 +121,18 @@ pub async fn get_block_by_number(url: &str, block: u64) -> Result<Block> {
 	}
 }
 
-pub fn generate_random_cells(max_rows: u16, max_cols: u16) -> Vec<Position> {
+pub fn generate_random_cells(max_rows: u16, max_cols: u16, cell_count: u32) -> Vec<Position> {
```

Comment:
Do we need some sort of sanity check for the cell count, ie in order not to overflow the multiplication?

Reply:
yeh this needs to be done!!